### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by jwilner**
### Review

@reviewer

### Summary

Use correct kustomize links following https://github.com/Kong/kubernetes-ingress-controller/pull/1682

### Reason

The kustomize instructions in the quickstart were incorrect following very recent changes 

### Testing

I've used it in local kustomize.

### CODE REVIEW

**Comment by lena-larionova** [comment]
Thanks for the submission, and nice catch!
It does look like this old directory (../deploy/manifests/..) is referenced in the docs in a few more places. 

This looks right to me, checked the links. @rainest @mflendrich - can you take a quick look? I can update the links globally if this is the desired location.

**Comment by rainest** [comment]
Those are the correct locations for the base files now. We're in kind of a weird situation since those currently since those build manifests that use KIC 2.x, but they're compatible with older Kong versions also as far as we know, so while 2.x is still in beta for a short while, we may as well just put them out there rather than trying to point back to older tags.

We do still place the final manifests we build ourselves under `deploy/single` and those should remain in place. Looking at what actually is in docs, we should make these changes:

```
deploy/manifests/enterprise-k8s -> config/variants/enterprise
deploy/manifests/base -> config/base
```

**Comment by lena-larionova** [comment]
Hi @jwilner  - based on @rainest 's comment above, can you please make the changes in the following files as well?
* https://github.com/jwilner/docs.konghq.com/blob/patch-2/app/kubernetes-ingress-controller/1.3.x/deployment/k4k8s-enterprise.md
* https://github.com/jwilner/docs.konghq.com/blob/patch-2/app/kubernetes-ingress-controller/1.3.x/deployment/k4k8s.md
* https://github.com/jwilner/docs.konghq.com/blob/patch-2/app/kubernetes-ingress-controller/1.3.x/concepts/deployment.md

**Comment by lena-larionova** [comment]
Hey @jwilner ! Could you apply the suggested changes? We have others mentioning this issue and would like to get the fix merged. 

If you prefer, I can make the changes, if you let me push to your fork.

**Comment by jwilner** [comment]
@lena-larionova, @rainest  hey, sorry for the delay, believe I've made the requested additional changes. 

All other files look like they're different versions?
```shell
rg -l deploy/manifests/
app/gateway-oss/1.4.x/kong-for-kubernetes/install.md
app/gateway-oss/2.0.x/kong-for-kubernetes/install.md
app/gateway-oss/2.4.x/kong-for-kubernetes/install.md
app/gateway-oss/2.3.x/kong-for-kubernetes/install.md
app/gateway-oss/2.2.x/kong-for-kubernetes/install.md
app/gateway-oss/1.5.x/kong-for-kubernetes/install.md
app/gateway-oss/2.1.x/kong-for-kubernetes/install.md
app/kubernetes-ingress-controller/1.0.x/concepts/deployment.md
app/kubernetes-ingress-controller/1.1.x/concepts/deployment.md
app/kubernetes-ingress-controller/1.2.x/concepts/deployment.md
app/kubernetes-ingress-controller/1.2.x/deployment/k4k8s.md
app/kubernetes-ingress-controller/1.1.x/deployment/k4k8s.md
app/kubernetes-ingress-controller/1.1.x/deployment/k4k8s-enterprise.md
app/kubernetes-ingress-controller/1.2.x/deployment/k4k8s-enterprise.md
app/kubernetes-ingress-controller/1.0.x/deployment/k4k8s.md
app/kubernetes-ingress-controller/1.0.x/deployment/k4k8s-enterprise.md
```

Also I think you _should_ have write access to my fork if you need it :)
image

File: app/kubernetes-ingress-controller/1.3.x/concepts/deployment.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ has the above permissions. The Ingress Controller Pod then has this
 necessary authentication and authorization tokens to communicate with the
 Kubernetes API-server.
 
-[rbac.yaml](https://github.com/Kong/kubernetes-ingress-controller/blob/main/deploy/manifests/base/rbac.yaml) contains the permissions
+[rbac.yaml](https://github.com/Kong/kubernetes-ingress-controller/blob/main/config/base/rbac.yaml) contains the permissions
```
Comment (lena-larionova) [inline app/kubernetes-ingress-controller/1.3.x/concepts/deployment.md:106]:
```suggestion
[rbac.yaml](https://github.com/Kong/kubernetes-ingress-controller/tree/main/config/rbac) contains the permissions
```
This link isn't working; I went looking for the new location, and it looks like there's way more that just one RBAC file now, so I linked it to the folder instead.

**Review by lena-larionova — COMMENTED** [review]
Thanks for making the changes! Just found one issue with one of the links.

> All other files look like they're different versions?

Yep, you can leave those alone. Updating just the latest version is enough.

**Review by lena-larionova — APPROVED** [review]
Fixed one of the links, LGTM! Thanks for your help :)

**Pull request merged**